src/datasources/entity-ds/queries/queryNodes.ts

```diff
@@ -1,5 +1,5 @@
-import { ClientDelegate } from "lib/client_delegate";
-import { OnmsNode } from "opennms/src/model/OnmsNode";
+import { ClientDelegate } from 'lib/client_delegate'
+import { OnmsNode } from 'opennms/src/model/OnmsNode'
```

Comment:
it would be nice if rando editor format cleanups were in a separate commit so it's easier to review the "real" changes

(or we should just do one big cleanup pass with prettier or something if we want to enforce different styles)

Reply:
Good point. Maybe format cleanups could be a separate commit. I've been updating files as I change them, haven't decided whether to do one big cleanup PR.